application/testing/CMakeLists.txt

```diff
@@ -658,6 +658,7 @@ f3d_test(NAME TestInteractionMisc DATA cow.vtp NO_BASELINE INTERACTION) #KK
 f3d_test(NAME TestInteractionCycleCell DATA waveletArrays.vti INTERACTION DEFAULT_LIGHTS) #VCCC
 f3d_test(NAME TestInteractionCycleComp DATA dragon.vtu INTERACTION DEFAULT_LIGHTS) #SYYYY
 f3d_test(NAME TestInteractionCycleScalars DATA dragon.vtu INTERACTION DEFAULT_LIGHTS) #BSSSS
+f3d_test(NAME TestInteractionCycleAnimation DATA InterpolationTest.glb INTERACTION DEFAULT_LIGHTS) #WWWWWWWWW;Space;Space;
```

Comment:
the log file seems to not have been added yet

Reply:
Okay, added! I didn't add it before because it has my unique file path URL on line 4: `/Users/doge/Desktop/f3d/src/testing/data/InterpolationTest.glb`

Reply:
That is definitely an issue, you should not use  "drop files" when recording, but load the file from the command line instead.